mozilla/kuma
# Bug 1197444: Move fonts to kuma/static

## Setup

`vagrant provision`

## Testing

1. Verify that fonts look good around the site (good pages to test are the [Demo Studio](https://developer-local.allizom.org/en-US/demos), the [Dev Derby](https://developer-local.allizom.org/en-US/demos/devderby), the [Promote page](https://developer-local.allizom.org/en-US/promote), and other pages that use uncommon fonts)
2. Enter [production mode](https://kuma.readthedocs.org/en/latest/development.html#production-assets)
3. Repeat step 1
